common/common-rest/src/main/java/org/apache/servicecomb/common/rest/codec/RestCodec.java

```diff
@@ -61,7 +64,11 @@ public static void argsToRest(Object[] args, RestOperationMeta restOperation,
       try {
         paramValues[idx] = param.getParamProcessor().getValue(request);
       } catch (Exception e) {
-        // Avoid information leak of user input.
+        // Avoid information leak of user input, and add option for debug use.
+        if (DynamicPropertyFactory.getInstance().getBooleanProperty(
+            RestConst.PRINT_CODEC_ERROR_MESSGAGE, false).get()) {
+          LOG.error("", e);
+        }
```

Comment:
I want to get the original exception information in the code. This is just printed out in the log. Can you wrap the exception as cause and throw it together in InvocationException?

Reply:
It's not possible doing that in java-chassis. "Exception" is a complicated object that cannot expressed by swagger, and the serialization/deserialization may couple consumer/provider.  

see:
https://docs.servicecomb.io/java-chassis/zh_CN/general-development/error-handling.html

Developers should not send unkown exceptions to consumers to avoid un-neccessary information leak and try to catch exceptions and return friendly error code and error mesasge is recommented.

Reply:
Hi, maybe we can set the raw exception to the cause of the InvocationException?
See the constructor `InvocationException(int statusCode, String reasonPhrase, Object errorData, Throwable cause)`

Reply:
This will have drawbacks. We can not expose internal exception trace to client(not recommented). And if we do serialize exception to the client, the client may be not possible to decode the exception.